docs/resources/script_module.md

```diff
@@ -30,13 +30,13 @@ resource "octopusdeploy_script_module" "example" {
 ### Required
 
 - `name` (String) The name of this resource.
-- `script` (Block Set, Min: 1, Max: 1) The script associated with this script module. (see [below for nested schema](#nestedblock--script))
 
 ### Optional
 
 - `description` (String) The description of this script module.
 - `id` (String) The unique ID for this resource.
-- `space_id` (String) The space ID associated with this resource.
+- `script` (Block List) The script associated with this script module. (see [below for nested schema](#nestedblock--script))
```

Comment:
This is still required at the validator level, just not the schema as Block Lists do not allow it. There's a number of resources with this problem which I'm planning on looking into this week.